Barotrauma/BarotraumaShared/SharedSource/Characters/Health/Afflictions/AfflictionHusk.cs

```diff
@@ -208,6 +211,30 @@ private IEnumerable<object> CreateAIHusk()
                 husk.Info.Character = husk;
                 husk.Info.TeamID = CharacterTeamType.None;
             }
+#if CLIENT
```

Comment:
There's unnecessary repetition here. I'd try to rewrite it so that the each logic statement is written once and only the parameters change. On the other hand, since the code blocks are meant for only either for client or for the server, would it make sense to move them from the shared class to dedicated client and server classes, since we do have those for AfflictionHusk? Perhaps using CreateAIHuskProjectSpecific methods, as has been the practice elsewhere in the code base?

Reply:
I cleaned up the code to make the client and server code blocks as small as possible. To be completely honest I'm not sure how to go about the other method you suggested since I'm very new to this code base and C# in general.